api/python/provisioner/commands/deploy.py

```diff
@@ -268,35 +281,26 @@ def _run_states(self, states_group: str, run_args: run_args_type):
         primary = self._primary_id()
         secondaries = f"not {primary}"
 
+        hw_states = [
```

Comment:
Network & firewall too comes here if we are making this special step.
Currently move this to **_system_** stage.